### DISCUSSION TITLE
CSS Componets

### DISCUSSION THREAD

**Discussion by jumalaw98** [SpaceyaTech/Team-Berlin#29] [Q&A] [answered]
## CSS
Which CSS library or framework do you, developers think of, so as we can handle the project, 
- Boostrap 
- Tailwind
- Chakra
- Bulma 
- or any other you can bring up

**Comment by daveclinton** [comment, marked as answer]
Chakra UI @jumalaw98

**Reply by jumalaw98** [reply]
Fine hope other person will be comfortable with this
